vnc.html

```diff
@@ -319,8 +319,7 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br />VNC</h1>
              on-screen keyboard. Let's hope Chrome implements the ime-mode
              style for example -->
         <textarea id="noVNC_keyboardinput" autocapitalize="off"
-            autocorrect="off" autocomplete="off" spellcheck="false"
-            mozactionhint="Enter" tabindex="-1"></textarea>
+            autocomplete="off" spellcheck="false" tabindex="-1"></textarea>
```

Comment:
I assume those attributes served a purpose at one point? Are we sure they're no longer needed.

Reply:
* mozactionhint - firefox only, the default is the enter-symbol anyway
* autocorrect - safari extension, has no effect now-a-days